docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -317,44 +335,190 @@ To enable this, we simply need to add `Contracts` type that we defined in the `c
 ```rust
 impl balances::Trait for Runtime {
     /// What to do if an account's free balance gets zeroed.
-    type OnFreeBalanceZero = (Contracts);
+    type OnFreeBalanceZero = Contracts;
     /* --snip-- */
 }
 ```
-
 Now, when the Balances pallet detects that the free balance of an account has reached zero, it calls the `on_free_balance_zero` function of the Contracts pallet.
 
+### Exposing The Contracts API
+
+We now need to enable calls to the contracts pallet. Not all of the pallets require using a specialized configuration to interact with them. However, the contracts pallet requires these API to call the sandboxed contracts.
```

Comment:
I don't know whether this is true, but myunderstanding, was that this RPC was just for doing dry-run contract calls and reading contract storage from offchain, and it is still possible to make actual on-chain calls without the RPC. If that is indeed the case, I think this sentence is misleading.

Reply:
Fair point it is part true but it requires more clarification. As you said it works to call the local node storage instead of making a transaction to get the data.